flake.nix

```diff
@@ -34,11 +34,17 @@
         [ "x86_64-linux" "aarch64-darwin" ]);
 
       makePkgs = { system, extraOverlays ? [ ], ... }@attrs:
-        import nixpkgs ({
+        let pkgs = import nixpkgs ({
           inherit system;
-          overlays = [ self.overlays.${system}.default ] ++ extraOverlays;
+          overlays = [ self.overlays.${system}.default ];
           config.allowUnfree = true;
         } // attrs);
+        in
+          /*
```

Comment:
comments are `#` in nix

Reply:
Both works, and `nixpkg-fmt` will make it look ugly for some reason
```nix
# abcd
  # xyz
```